datadog_monitor doesn't automatically restart the Agent (#596)
DataDog/chef-datadog: README.md

### Example

> This example enables the ElasticSearch integration by using the `datadog_monitor` resource. It provides the instance configuration (in this case: the url to connect to ElasticSearch) and set the `use_integration_template` flag to use the default configuration template. Also, it notifies the `service[datadog-agent]` resource in order to restart the Agent. If you use multiple times the resource `datadog_monitor` in your cookbook, it's recommended for speed purpose to restart the Agent only with the last call.

Reviewer: ```suggestion
This example enables the ElasticSearch integration by using the `datadog_monitor` resource. It provides the instance configuration (in this case: the url to connect to ElasticSearch) and sets the `use_integration_template` flag to use the default configuration template. Also, it notifies the `service[datadog-agent]` resource in order to restart the Agent.
```

(note: no need to use `notifies` only once: by default `notifies` is used with the `:delayed` option, which means that Chef will queue the `notifies` until the end of the Chef run, dedupe them, and actually restart the service only once at the end of the run. Pretty neat stuff)
